libs/domain/product/src/models/product.model.ts

```diff
@@ -41,6 +41,8 @@ export interface Product {
   mediaSet?: ProductMediaSet[];
   labels?: ProductLabel[];
   availability?: ProductAvailability;
+  discontinued?: boolean;
+  discontinuedNote?: string;
```

Comment:
should it support i18n messages?

Reply:
The api has only a single response, but when we request it with different language, it will respond a different message. No need to store various mesaages for each locale.